sql/src/main/java/io/crate/executor/transport/FirstColumnConsumer.java

```diff
@@ -36,9 +36,9 @@
 import java.util.stream.Collector;
 
 /**
- * Collector to retrieve all values of a single column in a query.
+ * Collector to retrieve all values of the first column in a query.
```

Comment:
sorry did not see it in the first place: but it takes the first column of each row. (not query)